<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEssayPaper</class>
 <widget class="QWidget" name="NewEssayPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tWPaperCreation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>641</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add qustions</string>
    </attribute>
    <widget class="QPushButton" name="pBClose">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>560</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBEssay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>1111</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Add Essay Question</string>
     </property>
     <widget class="QLabel" name="lEssayQusesion">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Question :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEEssayQuestion">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>711</width>
        <height>61</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pBAddEssayQuestion">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Essay Question</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEEssayMarks">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Marks For Question</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Create Answers</string>
    </attribute>
    <widget class="QGroupBox" name="gBHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBQuestions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>1111</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QFrame" name="fQuestion1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pBCClose">
     <property name="geometry">
      <rect>
       <x>984</x>
       <y>580</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save All And Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBPrivious">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>582</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Privious</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBNext">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>582</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="lPageNo">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>590</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Edit Questions</string>
    </attribute>
    <widget class="QGroupBox" name="gBEHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBEQuestions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>1111</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QFrame" name="fEQuestion1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lEPageNo">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>590</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBEClose">
     <property name="geometry">
      <rect>
       <x>984</x>
       <y>580</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save All And Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBENext">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>580</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBEPrivious">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>580</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Privious</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
